kobo/client/commands/cmd_create_worker.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+
+import sys
+from six.moves.xmlrpc_client import Fault
+
+from kobo.client import ClientCommand
+
+
+class Create_Worker(ClientCommand):
+    """create a worker"""
+    enabled = True
+    admin = True
+
+
+    def options(self):
+        self.parser.usage = "%%prog %s [--create] worker_name [worker_name]" % self.normalized_name
+
+        self.parser.add_option(
+            "--create",
+            default=False,
+            action="store_true",
+            help="create worker"
+        )
+
+
+    def run(self, *args, **kwargs):
+        create = kwargs.pop("create", False)
+
+        if len(args) == 0:
+            self.parser.error("No worker name specified.")
+
+        username = kwargs.pop("username", None)
+        password = kwargs.pop("password", None)
+        hub = kwargs.pop("hub", None)
+        workers = args
+
+        self.set_hub(username, password, hub)
+        for worker in workers:
+            try:
+                new_worker = self.hub.client.create_worker(worker)
+                print(new_worker)
+            except Fault as ex:
+                sys.stderr.write("%s\n" % ex.faultString)
```

Comment:
This avoids pretty-printing of tracebacks sent over XML-RPC.  Use the following code instead, please: 
```suggestion
                print(repr(ex), file=sys.stderr)
```
The rest of admin commands should by fixed by https://github.com/release-engineering/kobo/pull/208.